clients/spark/build.sbt

```diff
@@ -63,7 +63,7 @@ libraryDependencies ++= Seq(
   "com.azure" % "azure-storage-blob" % "12.9.0",
   "com.azure" % "azure-storage-blob-batch" % "12.7.0",
   "com.azure" % "azure-identity" % "1.2.0",
-  "com.amazonaws" % "aws-java-sdk-bundle" % "1.12.194" % "provided",
+  "com.amazonaws" % "aws-java-sdk-bundle" % "1.12.569" % "provided",
```

Comment:
But what if it's the previous version? Doesn't this version mean that the AWS SDK used is V2? (it might not be a problem if the older hadoop version works with it, but I think that even the paths from which you import classes is different)